src/core/settings.c

```diff
@@ -1615,6 +1620,7 @@ QuicSettingsSettingsToInternal(
     SETTING_COPY_TO_INTERNAL(DatagramReceiveEnabled, Settings, InternalSettings);
     SETTING_COPY_TO_INTERNAL(ServerResumptionLevel, Settings, InternalSettings);
     SETTING_COPY_TO_INTERNAL(GreaseQuicBitEnabled, Settings, InternalSettings);
+    SETTING_COPY_TO_INTERNAL(ReliableResetEnabled, Settings, InternalSettings);
```

Comment:
You had it right before, undo this change.

Reply:
done and pushed